### DISCUSSION TITLE
新手摸瞎中，使用.\env-helper.ps1虽然没报错，但还是出现了安装过程的问题

### DISCUSSION THREAD

**Discussion by tgfydzs** [zepp-health/zeppos-forum#323] [Q&A] [answered]
具体代码为：

[node] v20.11.1 ✓
[npm] 10.2.4 ✓
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'C:\Users\13261\AppData\Roaming\npm\node_modules\@zeppos\zeus-cli\bin\main.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.11.1
[zeus]  ✓
[taptun] Network Interface Name: tun, IPAddress: 1 ✓

中间的error部分有没有影响？

**Comment by tgfydzs** [comment]
不管开不开代理在 reify:rxjs: timing reifyNode:node_modules/@zeppos/zeus-cli Completed in 2223ms总是最慢。。。。
这还是用的npm i @zeppos/zeus-cli -g --registry "https://registry.npmmirror.com"，这是官方说中国大陆不好用的话试试这个，依然不行

**Comment by MyoungXUE** [comment, marked as answer]
Hello @tgfydzs, it seems that the installation of zeus-cli encountered a problem, it is recommended to use `npm uninstall @zeppos/zeus-cli -g` and then refer to this document to reinstall https://docs.zepp.com/docs/guides/faq/env-setup/

It is recommended to use English for discussion in the Github discussion area, which will help developers from all over the world.
